src/lib/dates.ts

```diff
@@ -64,6 +64,11 @@ export const DEFAULT_DURATIONS: DurationOption[] = [
   { label: 'Days', value: 'day' },
 ];
 
+export const DEFAULT_RELATIVE_VARIABLES = [
+  { identifier: 'today', value: '$$NOW', label: 'TODAY' },
```

Comment:
Another todo would be to adapt this "NOW" to each backend ($$NOW will not work in pandas or snowflake)